applications/nrf5340_audio/src/utils/nrf5340_audio_dk.c

```diff
@@ -20,6 +23,39 @@ LOG_MODULE_REGISTER(nrf5340_audio_dk, CONFIG_MODULE_NRF5340_AUDIO_DK_LOG_LEVEL);
 
 static struct board_version board_rev;
 
+static int hfclock_init(void)
```

Comment:
Could we have a word on how this differs from the previous solution @rick1082 ?

Reply:
Now we don't need to start HFCLK all the time, since that was prepared for PCFT.
So we can remove those part now, probably we can also get some benefits for power consumption.